packages/opentelemetry-core/src/common/time.ts

```diff
@@ -140,6 +140,14 @@ export function hrTimeToMicroseconds(time: api.HrTime): number {
   return time[0] * 1e6 + time[1] / 1e3;
 }
 
+/**
+ * Convert hrTime to seconds.
+ * @param time
```

Comment:
The `@param` tag isn't contributing much here. I think it can be removed as the type itself is pretty obvious. May want to add a note that precision is lost in the conversion.

Reply:
Then you'd make things inconsistent with the other utilities.